_layouts/default.html

```diff
@@ -3,9 +3,9 @@
   {% include head.html %}
   <body>
     {% include header.html %}
-    <div role="main" class="container">
+
       {{ content }}
-    </div>
+
     {% include footer.html %}
```

Comment:
I think we were using the container as a wrapper to centre everything. If you check the secondary pages things are left aligning. I'm not sure if we want to move away from this?